include/xen/generic.h

```diff
@@ -11,4 +11,27 @@
 #define XEN_PAGE_SIZE		4096
 #define XEN_PAGE_SHIFT		12
 
+#define PFN_UP(x)		(unsigned long)(((x) + XEN_PAGE_SIZE-1) >> XEN_PAGE_SHIFT)
```

Comment:
The name of the PFN* helpers are quite generic. I would suggest to prefix with XEN_* to avoid any future clash.

Reply:
Agree, fixed.